node_modules/.package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
there shouldn't be a node_modules folder in the top level directory.  Did you run an npm install command while you were outside of the client directory? Please remove